test/base58_test.exs

```diff
@@ -0,0 +1,88 @@
+defmodule Base58Test do
```

Comment:
Don't know why GitHub thinks this is a new file, I just renamed it.
`.then` added a few new tests. 📝